ci-operator/step-registry/acm/policies/openshift-plus/acm-policies-openshift-plus-commands.sh

```
@@ -13,36 +13,86 @@ git clone https://github.com/stolostron/policy-collection.git
+QUAY_RETRIES=20
+QUAY_FOUND=false
+for try in $(seq "${QUAY_RETRIES}"); do
+  # Check multiple possible namespaces where Quay might be deployed
+  for ns in quay openshift-quay quay-enterprise; do
+    if oc get route -n $ns quay-registry &>/dev/null 2>&1; then
+      QUAY_ROUTE=$(oc get route -n $ns quay-registry -o jsonpath='{.spec.host}' 2>/dev/null || echo "")
+      if [ -n "$QUAY_ROUTE" ]; then
+        echo "Quay registry route found in namespace '$ns': $QUAY_ROUTE"
+        # Give Quay additional time to become fully operational
+        sleep 60
+        QUAY_FOUND=true
```

Shellcheck (0.11.0)

[warning] QUAY_FOUND appears unused. Verify use (or export if used externally).

(SC2034)